x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/ml_inference_logic.ts

```diff
@@ -34,10 +34,10 @@ import {
   MappingsApiLogic,
 } from '../../../../api/mappings/mappings_logic';
 import {
-  TrainedModel,
-  TrainedModelsApiLogicActions,
-  TrainedModelsApiLogic,
-} from '../../../../api/ml_models/ml_trained_models_logic';
+  CachedFetchModelsApiLogic,
+  CachedFetchModlesApiLogicActions,
+} from '../../../../api/ml_models/cached_fetch_models_api_logic';
+import { FetchModelsApiResponse } from '../../../../api/ml_models/fetch_models_api_logic';
```

Comment:
nit: I might add an export of this to `api/ml_models/cached_fetch_models_api_logic` so we only have to rely on that api logic file and consumers of `cached_fetch_models_api_logic` don't need to import `fetch_models_api_logic`

Reply:
@TattdCodeMonkey Did you mean like [this](https://github.com/elastic/kibana/pull/176257/commits/26ff28765e32320d5297b5d4b79e102589fa001e)?